tooling/debugger/src/context.rs

```diff
@@ -692,9 +690,17 @@ mod tests {
                 },
                 BrilligOpcode::Stop { return_data_offset: 0, return_data_size: 0 },
             ],
-            predicate: None,
         };
-        let opcodes = vec![Opcode::Brillig(brillig_opcodes)];
+        let opcodes = vec![Opcode::BrilligCall {
+            id: 0,
+            inputs: vec![BrilligInputs::Single(Expression {
+                linear_combinations: vec![(fe_1, w_x)],
+                ..Expression::default()
+            })],
+            outputs: vec![],
+            predicate: None,
+        }];
+        let brillig_funcs = &vec![brillig_bytecode];
```

Comment:
```suggestion
        let brillig_functions = &vec![brillig_bytecode];
```
nit to avoid the spell check warning

Reply:
I see this name is what was there before so we can just merge this